scripts/xnat/check_new_sessions

```diff
@@ -436,27 +436,24 @@ def frame_check(eid, manufacturer, label):
         for f in frames:
             if f[0] in scan_type_dictionary['GE MEDICAL SYSTEMS']:
                 if frame_check_ge[f[0]] != f[1]:
-                    error.append({'field': 'Frames',
-                                  'scan_type': f[0],
+                    error.append({'scan_type': f[0],
                                   'frames': f[1],
                                   'expected_frames': frame_check_ge[f[0]]})
     elif manufacturer == 'SIEMENS':
         for f in frames:
-            if f[0] in scan_type_dictionary['SIEMENS']:
-                if f[0] == 'ncanda-grefieldmap-v1':
-                    if f[1] in frame_check_siemens[f[0]]:
-                        f[1] = f[1]
-                    else:
-                        error.append({'field': 'Frames',
-                                      'scan_type': f[0],
-                                      'frames': f[1],
-                                      'expected_frames': frame_check_ge[f[0]]})
-                else:
-                    if frame_check_siemens[f[0]] != f[1]:
-                        error.append({'field': 'Frames',
-                                      'scan_type': f[0],
-                                      'frames': f[1],
-                                      'expected_frames': frame_check_ge[f[0]]})
+            if f[0] not in scan_type_dictionary['SIEMENS']:
+                continue 
+            if f[0] == 'ncanda-dti6b500pepolar-v1':
+                # Special case, this frame can contain either '7' or '8' values
```

Comment:
part of the issue is related to https://github.com/sibis-platform/ncanda-operations/issues/1937

I think when that 8th frame was included it doesn't have any of the "dwi" parameters, thus the check fails when it checks the XML sidecare file I showed you for a 'dwi' element.

I think I would check for 7, and if 8, then create an error and avoid even trying to convert to nifti